### PR TITLE
mavproxy_cmdlong: attitude add angular rates & mask

### PR DESCRIPTION
**Pull request by hendjoshsr71**
This updates the attitude command so you send the typemask and angular velocities.

image

### CODE REVIEW

File: MAVProxy/modules/mavproxy_cmdlong.py [hunk 1]
```diff
@@ -251,29 +251,34 @@ def cmd_position(self, args):
                                       0, 0)     # yaw, yaw rate
 
     def cmd_attitude(self, args):
-        '''attitude q0 q1 q2 q3 thrust'''
-        if len(args) != 5:
-            print("Usage: attitude q0 q1 q2 q3 thrust (0~1)")
+        '''attitude mask q0 q1 q2 q3 roll_rate pitch_rate yaw_rate thrust'''
+        if len(args) != 9:
+            print("Usage: attitude mask q0 q1 q2 q3 roll_rate pitch_rate yaw_rate thrust (0~1)")
```
Comment (tridge) [inline MAVProxy/modules/mavproxy_cmdlong.py:256]: units, so roll_rate(deg/s)

File: MAVProxy/modules/mavproxy_cmdlong.py [hunk 2]
```diff
@@ -251,29 +251,34 @@ def cmd_position(self, args):
                                       0, 0)     # yaw, yaw rate
 
     def cmd_attitude(self, args):
-        '''attitude q0 q1 q2 q3 thrust'''
```
Comment (tridge) [inline MAVProxy/modules/mavproxy_cmdlong.py:254]: support the old 5 arg version

**Review by tridge — CHANGES_REQUESTED** [review]

File: MAVProxy/modules/mavproxy_cmdlong.py (hunk 1 above)
Comment (tridge) [reply MAVProxy/modules/mavproxy_cmdlong.py:256]: or 2nd line of usage saying rates are in deg/sec
and quaternion convention

File: MAVProxy/modules/mavproxy_cmdlong.py (hunk 2 above)
Comment (hendjoshsr71) [reply MAVProxy/modules/mavproxy_cmdlong.py:256]: done

**Review by tridge — APPROVED** [review]

**Pull request merged**